apps/client/src/pages/chat/page.tsx

```diff
@@ -0,0 +1,118 @@
+'use client';
+import { IMsgDataTypes } from '@repo/shared-types/src/types';
+import React, { useEffect, useState } from 'react';
+
+import style from './page.module.css';
+
+const ChatPage = ({
+  socket,
+  username,
+  roomId,
+}: {
+  socket: any;
```

Comment:
import { Socket } from 'socket.io-client';
```suggestion
  socket: Socket;
```